draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1402,6 +1402,45 @@ Note that it is possible for a certificate to be logged more than once. If that
 is the case, the log MAY return more than one entry index. If the certificate is
 present in the log, then the log MUST return at least one entry index.
 
+## Retrieve Signed Tree Heads between Two Times
+
+GET https://\<log server>/ct/v2/get-sths
```

Comment:
Are you going to say something about this end-point being optional?
Or were you planning to leave that until the decision that it should be optional is made?